csp-guru/datapackage.json

```diff
@@ -1,15 +1,167 @@
+                        "name": "Total_cost",
+                        "description": "in thousand local currency"
+                    },
+                    {
+                        "name": "Specific_investment_cost_2015USD",
+                        "description": "investment cost per MW [2015 USD], discounted"
+                    },
+                    {
+                        "name": "Specific_investment_cost_2015EUR",
+                        "description": "investment cost per MW [2015 EURO], discounted"
+                    },
+                    {
+                        "name": "Developer",
+                        "description": "company that developed the project"
+                    },
+                    {
+                        "name": "HCE_manufacturer",
+                        "description": "heat collecting element (HCE) manufacturer"
+                    },
+                    {
+                        "name": "SCA_manufacturer",
+                        "description": "solar collector assemblies (SCA) manufacturer"
+                    },
+                    {
+                        "name": "Financed_through",
+                        "description": "how the owner makes money from the generated electricity: PPA, FiT, spot sales"
+                    },
+                    {
+                        "name": "Remuneration",
+                        "description": "indicates the level of FiT or PPA [ct/kWh], local currency"
+                    },
+                    {
+                        "name": "Remuneration_2015USD",
+                        "description": "indicates the level of FiT or PPA [ct/kWh], 2015 USD "
+                    },
+                    {
+                        "name": "Remuneration_2015EUR",
+                        "description": "indicates the level of FiT or PPA in [ct/kWh], 2015 EURO "
+                    },
+                    {
+                        "name": "LCOE_5_2015USD",
+                        "description": "levelized cost of electricity with 5 years payback [ct/kWh], 2015 USD discounted"
+                    },
+                    {
+                        "name": "LCOE_10_2015USD",
+                        "description": "levelized cost of electricity with 10 years payback [ct/kWh], 2015 USD discounted"
+                    },
+                    {
+                        "name": "LCOE_5_2015EUR",
+                        "description": "levelized cost of electricity with 5 years payback [ct/kWh], 2015 EURO discounted"
+                    },
+                    {
+                        "name": "LCOE_10_2015EUR",
+                        "description": "levelized cost of electricity with 10 years payback [ct/kWh], 2015 EURO discounted"
+                    },
+                    {
+                        "name": "Other_support",
+                        "description": "financial assistence to project developer like tax credits, cheap credits, etc."
```

Comment:
"description": "financial assistance to project developer: tax credits, cheap credits, etc."